packages/curve-ui-kit/src/lib/logging.ts

```diff
@@ -101,7 +101,7 @@ export function log(key: LogKey, status?: LogStatus | unknown, ...args: unknown[
 
   const hasDefinedStatus = status && Object.values(LogStatus).includes(status as LogStatus)
   const [formattedKeyString, keyStyles] = formatKeyArray(keyArray)
-  const restArgs = hasDefinedStatus ? args : [status, ...args]
+  const restArgs = (hasDefinedStatus ? args : [status, ...args]).filter(Boolean)
```

Comment:
I don't think we should filter out `0`
```suggestion
  const restArgs = (hasDefinedStatus ? args : [status, ...args]).filter(a => a != null)
```